lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -56,6 +63,10 @@ export async function updateArtifacts({
     const distributionUrl = getDistributionUrl(newPackageFileContent);
     if (distributionUrl) {
       cmd += ` --gradle-distribution-url ${distributionUrl}`;
+      if (newPackageFileContent.includes('distributionSha256Sum=')) {
+        const checksum = await getDistributionChecksum(distributionUrl);
```

Comment:
Do you prefer to totally fail the artifacts updating if the checksum file is missing, or skip it and add a synthetic artifacts error message?

Reply:
If we don't update the checksum, gradle run will fail on ci, because it checks the sum on *next* run, not on self update